ask/alexa_io.py

```diff
@@ -192,11 +210,18 @@ def _handler(func):
 
         return _handler
 
-    def route_request(self, request_json, metadata=None):
+    def route_request(self, request_json, metadata=None, app_ids=None):
```

Comment:
It seems to me that having the app_ids as a construction parameter for the 'alexa' makes more sense than passing it into a parameter for route_request

Reply:
i agree, should i add it to the constructor somehow? or do `alexa.app_ids = [...]` in lambda_handler?